textrender.h

```diff
@@ -29,6 +29,7 @@ class TextRender : public QQuickItem
     Q_PROPERTY(DragMode dragMode READ dragMode WRITE setDragMode NOTIFY dragModeChanged)
     Q_PROPERTY(QQuickItem* contentItem READ contentItem WRITE setContentItem NOTIFY contentItemChanged)
     Q_PROPERTY(QFont font READ font WRITE setFont NOTIFY fontChanged)
+    Q_PROPERTY(QSizeF cellSize READ cellSize NOTIFY cellSizeChanged)
```

Comment:
Didn't change now, but should have integer sized cells?

Reply:
Unfortunately not, that will break with a good number of fonts, IIRC

Reply:
Out of curiosity, in what way? A glyph should always fit into rounded up cell, no?

Reply:
It's been a while since I was working on this bit, but IIRC the problem is less with the font (because yeah, you'll generally have a little padding around the visual glyph), but with the cursor position. The cursor is positioned based on a multiple of the cell size, so the rounding errors accumulate and lead to the cursor position being increasingly incorrect the further away from the origin you go.

Reply:
Sounds maybe something like not consistently using integer vs floating point sizes. Could maybe later check if it would be possible to just use integers and have things aligned to pixels.